### PR TITLE
Update docs structure

### PR DESCRIPTION
**Pull request by tayarani**
# Changes

- Update README.md and add place holder docs
-
-

/kind <kind>

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
-# TODO Repository Name
+# Knative Specs Repository
 
-_TODO(leads): please fill in this section_
+This repository contains the specifications for Knative components, along with
+policies and procedures around conformance and certifiation.
```
Comment (grantr) [inline README.md:4]: typo nit: certification

**Review by grantr — COMMENTED** [review]
Nice work @tayarani. Was this layout inspired by an existing conformance effort?

File: README.md (hunk 1 above)
Comment (tayarani) [reply README.md]: fixed

**Comment by tayarani** [comment]
Yes, it was mostly inspired by https://github.com/cncf/k8s-conformance

**Review by grantr — APPROVED** [review]
/lgtm

**Pull request merged**